src/OmniSharp.Roslyn.CSharp/Services/Completion/CompletionOptionsProvider.cs

```diff
@@ -15,9 +15,9 @@ public class CompletionOptionsProvider : IWorkspaceOptionsProvider
         public int Order => 0;
 
         public OptionSet Process(OptionSet currentOptionSet, OmniSharpOptions omniSharpOptions, IOmniSharpEnvironment omnisharpEnvironment)
-            => currentOptionSet.WithChangedOption(
+            => currentOptionSet/*.WithChangedOption(
```

Comment:
@tmat: do we need to make a new API for Omnisharp so they can call into the completion service and specify the import completion flag? I guess this worked before by taking the value into an OptionSet here, and that was passed into  OmniSharpCompletionService.GetCompletionsAsync in CompletionService.cs; we didn't expose any way to pass your new struct through, so I'm guessing we need to revise that?

Reply:
Looking now.

Reply:
This has now all been fixed up in #2306.